packages/docusaurus-utils/src/__tests__/gitUtils.test.ts

```diff
@@ -136,4 +137,36 @@ describe('getFileCommitDate', () => {
       /Failed to retrieve git history for ".*nonexistent.txt" because the file does not exist./,
     );
   });
+
+  it('temporary created file that is not tracked by git', async () => {
```

Comment:
we are testing a feature of `../lastUpdateUtils`, so I'd prefer to have this test in `lastUpdateUtils.test.ts` instead of here